utils.sh

```diff
@@ -157,6 +157,22 @@ function docker_push() {
   parallel docker push {} ::: $queue
 }
 
+function bucket_push() {
+  [ "$ImageBucketPrefix" == "" ] && echo "nothing to do" && return
```

Comment:
We should capture this condition in the docs, e.g. add to the readme.md a line saying that images will be saved as tarballs if $ImageBucketPrefix is set.